### DISCUSSION TITLE
App window disappears after disconnecting from secondary monitor and changing from maximised to normal

### DISCUSSION THREAD

**Discussion by georgd** [Zettlr/Zettlr#2529] [General]
I was using Zettlr as a maximised window while working with multiple monitors attached to the laptop. After detaching the monitors I continued in maximised window first. Later I wanted to see the Zettlr window besides another and unmaximised it. The window instantly moved out of the laptop screen and can’t be retrieved.

This was on Windows 10 with the nightly from 2021-09-27.

Has anybody experienced that yet?

**Comment by canpolat** [comment]
This is something I experience on Windows 10 with several different applications. I had this problem just yesterday with Edge (Microsoft browser). So, I guess this problem is not specific to Zettlr.

**Reply by georgd** [reply]
Yes, you’re certainly right.

**Comment by nathanlesage** [comment]
Someone on our Discord mentioned that you can focus the window (using the task bar icon) and use Win+ArrowKeys to move it around. Or something like that. There is a solution apparently.

**Reply by georgd** [reply]
Oh my, too many channels. I thought I read that somewhere but couldn’t find the right place.

Thank you, that works.
